app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,7 @@
 
   <body>
     <%= render "shared/header" %>
+    <%= flash[:success] %>
```

Comment:
If you want to display your flash messages here, please render your _flash_messages partial file.